src/pyodide/python-entrypoint-helper.ts

```diff
@@ -176,11 +163,12 @@ try {
   if (!IS_TRACING) {
     if (IS_WORKERD) {
       // If we're in workerd, we have to do setupPackages in the IoContext, so don't start it yet.
+      //
+      // Also, not doing this here seems to cause @workerd//src/cloudflare/internal/test/d1:python-d1-api-test
+      // to fail under --config=asan.
+      //
       // TODO: fix this.
       await getPyodide();
```

Comment:
If I remove this call then some of our workerd tests fail under asan.

https://paste.cfdata.org/GdKtTZYLiz6LktnWP5gP

Hood mentioned this may be related to the isolate pool changes. @danlapid any ideas what might be going on here?

Reply:
Had to remove it again. No way around it. I'll probably disable these tests under asan for now but we should fix this asap, this asan issue is likely lurking there right now and just isn't being triggered.

Reply:
Pool code hasn't been merged yet so I don't see how it might be related